### DISCUSSION TITLE
Serial starting Lights by Group

### DISCUSSION THREAD

**Discussion by BuchbergerFlorian** [andesse/hue-clip-api.node-red-flows#40] [Q&A] [answered]
Hi Andreas. Your code works really great, thank you for it!
I still have a question. I created a room with 6 spots and can turn them on and off with no problems. As you suggested, I turn on the entire group with a scene.
However, when I turn on the group via your API, all the spots turn on at different speeds. When I switch on the room via the HUE app, they all start up in parallel and at the same speed.
Do you know what could be the reason for this? I would be grateful if you could help me there.
Lg Florian

**Comment by andesse** [comment, marked as answer]
Hi Florian,

are you recalling a scene in the app, or do you use the „room on function“?

I have +10 lamps in my livingroom, and whatever I recall the scene via node-red or app, my lights turn on at different speeds. 

The bridge does still send separate light commands when recalling a scene and due API limitations of sending 5 light commands per second, having more lights than 4-5 these will turn on at different speeds. 

When you turn on/off a group it’s faster, cause it’s send as a group  command, but the lights will turn on at last state. 

That’s what I think is the reason in my understanding of the API.

But anyway, great that it works!

Grüße 
Andreas

**Comment by BuchbergerFlorian** [comment]
Thank you Andreas for your quick feedback.
I use the "Room On" switch/function in the app. Since everything works without problems.
In NodeRed I use the "Recall a Scene" function and there are different switch-on speeds. I will try not to switch with a scene but to use the "Room On" function and compare the switch-on behavior.

**Comment by andesse** [comment]
@BuchbergerFlorian 

hat es so nun geklappt? 
Hier mal die API Referenz was alles per Gruppe geschaltet werden kann:

Properties

type: (grouped_light)
Type of the supported resources

on: (object)
Joined on control & aggregated on state.| – “on” is true if any light in the group is on

on: (boolean)
On/Off state of the light on=true, off=false

dimming: (object)
Joined dimming control – “dimming.brightness” contains average brightness of group containing turned-on lights only.

brightness: (number – maximum: 100)
Brightness percentage. value cannot be 0, writing 0 changes it to lowest possible brightness

dimming_delta: (object)
action: required(one of up, down, stop)
brightness_delta: (number – maximum: 100)
Brightness percentage of full-scale increase delta to current dimlevel. Clip at Max-level or Min-level.

color_temperature: (object)
Joined color temperature control

mirek: (integer – minimum: 153 – maximum: 500)
color temperature in mirek or null when the light color is not in the ct spectrum

color_temperature_delta: (object)
action: required(one of up, down, stop)
mirek_delta: (integer – maximum: 347)
Mirek delta to current mirek. Clip at mirek_minimum and mirek_maximum of mirek_schema.

color: (object)
Joined color control

xy: (object)
CIE XY gamut position

x: required(number – minimum: 0 – maximum: 1)
X position in color gamut

y: required(number – minimum: 0 – maximum: 1)
Y position in color gamut

alert: (object)
Joined alert control

action: required(breathe)
dynamics: (object)
duration: (integer)
Duration of a light transition or timed effects in ms.

**Reply by BuchbergerFlorian** [reply]
Hallo Andreas.
Ich habs nochmal ausprobiert. Wenn du mit NodeRed die Funktion "Room or Zone on" aktivierst, dann funktioniert es tadellos -> alle Lampen starten synchron und dimmen gleichmäßig auf ihr maximum.

**Reply by andesse** [reply]
> Hallo Andreas.
> 
> Ich habs nochmal ausprobiert. Wenn du mit NodeRed die Funktion "Room or Zone on" aktivierst, dann funktioniert es tadellos -> alle Lampen starten synchron und dimmen gleichmäßig auf ihr maximum.

Klasse. Für mich hat dies keine Anwendung, da ich abhängig von der Tageszeit andere Szenen schalte. Ich habe 18 Lampen im WZ 

Freut mich das es wie gewünscht so funktioniert :)
